### ISSUE TITLE
Icon Request: fa-explosion

### ISSUE THREAD

**Issue by cgilbu** [FortAwesome/Font-Awesome#5479]
Google "explosion icon" to get a visual on it. Comes in handy when dealing with emergency situations or preparedness.

**Comment by Arrow7000** [comment]
Any update on this?

Thanks

**Comment by Atulin** [comment]
Bumping it two years after original suggestion has been made. I could really use that icon.

**Comment by tinwe** [comment]
+1
If generic enough, this would even be great as a "hit" icon.
Make this happen.

**Comment by shaedrich** [comment]
Something like that: 
![image](https://user-images.githubusercontent.com/11225821/57005948-c963dc80-6b88-11e9-86d9-c1994eb35749.png)
Basically comic style

**Comment by VaniiiC** [comment]
any updates to this?

greetings.

**Comment by shaedrich** [comment]
> Something like that:
> ![image](https://user-images.githubusercontent.com/11225821/57005948-c963dc80-6b88-11e9-86d9-c1994eb35749.png)
> Basically comic style

Or with debris:
![image](https://user-images.githubusercontent.com/11225821/110213226-5e2f7380-7e9f-11eb-92d1-ef65257b0080.png)

**Comment by briandrum** [comment]
Looking for this as well. Basically the equivalent of [the collision emoji](https://emojipedia.org/collision/) 💥

**Comment by tinwe** [comment]
Yes, both regular explosion as well as with debris would be helpful

**Comment by sensibleworld** [comment]
https://fontawesome.com/search?q=explosion&s=solid%2Cbrands
